### ISSUE TITLE
Property Sets missing issue

### ISSUE THREAD

**Issue by SoftwareMechanic** [Creoox/creoox-ifc2gltfcxconverter#2]
Hello,
I was doing some testing, and found that: when 2 objects or more share an identical property set (same property names, same property values) sometimes it assign that PSET only once to one object, instead of assigning it to all relative objects.
Maybe this depends by the originary software which exported the IFC.
Here I attached a txt file (which is an IFC just with a different extension in order to be able to upload it here).
you should just check the columns data once converted.
On other IFC viewers I can find the 'WBS' pSet on every column.
on the file converted (or just in the json exported) I can find it only on one column per unique pSet.

this happened also with other files and other property sets in the same situation.

[ModellTest.txt](https://github.com/Creoox/creoox-ifc2gltfcxconverter/files/13990657/ModellTest.txt)

**Comment by ifcapps** [comment]
This issue is fixed in latest release: https://github.com/Creoox/creoox-ifc2gltfcxconverter/releases/tag/3.0.9
Please check and confirm or report if problems persist.

**Comment by SoftwareMechanic** [comment]
Thank you so much for the prompt response again!
I do appreciate a lot!

I confirm now the converter is working as expected 😄 

for what concerns the properties, I would like to help you out so I try to explain the current situation:
here is the metadata json file I got from the converter from the same file I shared before: 
[ModellostrutturaleWBS.json](https://github.com/Creoox/creoox-ifc2gltfcxconverter/files/14021285/ModellostrutturaleWBS.json)

Inside it you can check for example the following IFC Element GUID: 3mnEEKpjDAje5hLLhNJ$xg, it has only one property set related to it (pset id = 0bKhjE8iLEfh9M21Ivhw3K) which is BaseQuantities pset.

But it should have more psets, which are shown in the following image: 
![immagine](https://github.com/Creoox/creoox-ifc2gltfcxconverter/assets/53627894/68e29809-8111-4cc1-81a6-36c1a5b69c25)


I am currently using xeokit as SDK for the viewer and it's not even showing BaseQuantities pset ( but that's another issue I think I have to open in their repository).

Let me know if you need more detail about this!
Thank you again ✌️

**Comment by ifcapps** [comment]
I confirm there was a bug in property set export.
Now fixed in version 3.0.11
Please check and confirm or report if problems persist.

BaseQuantities should appear just like the property sets.
Here are the Psets and quantities in a gltf viewer (prior to converting to xkt):
![Screenshot 2024-01-23 185712](https://github.com/Creoox/creoox-ifc2gltfcxconverter/assets/107111242/89074919-3609-4cbe-843b-d298de14e1d2)

**Comment by SoftwareMechanic** [comment]
Amazing!
Now works PERFECTLY!

Thank you for the prompt update!
I appreciate a lot your effort.
